src/lib/card.js

```diff
@@ -11,13 +11,13 @@ export default function createCard(card) {
   )
   cardSection.append(bookmark)
 
-  if (card.isBookmarked) {
-    bookmark.classList.remove('far')
-    bookmark.classList.add('fas')
-  } else {
-    bookmark.classList.remove('fas')
-    bookmark.classList.add('far')
-  }
+  // if (card.isBookmarked) {
```

Comment:
You can remove this commented code. Don't worry it won't be gone, it will stay always in a commit ;)